src/main/java/com/suse/salt/netapi/calls/modules/Test.java

```diff
@@ -138,6 +138,12 @@ public VersionInformation(Map<String, String> salt,
         return PING;
     }
 
+    public static LocalCall<Boolean> ping(Optional<Integer> timeout,
```

Comment:
i don't think we should make `ping` a special case here if those parameters are supported on every function. We should rather do something along the lines of `withMetadata`.